test/powershell/Modules/Microsoft.Powershell.Host/Start-Transcript.Tests.ps1

```diff
@@ -129,6 +140,35 @@ Describe "Start-Transcript, Stop-Transcript tests" -tags "CI" {
         # Nothing should have been returned by the FileSystemWatcher
         Receive-Job $job | Should -Be $null
     }
+    It "Should keep the existing file's encoding if the 'Append' parameter is used"{
+        $transcriptFilePath = Join-Path $TestDrive ([System.IO.Path]::GetRandomFileName())
+        "Text" | Out-File -Encoding unicode $transcriptFilePath
+
+        Start-Transcript -Append -Path $transcriptFilePath
+        Get-Date | Out-Null
+        Stop-Transcript
+
+        GetFileEncoding $transcriptFilePath | Should -Be 'utf-16'
```

Comment:
Please use BeExactly for strings here and below:
```suggestion
        GetFileEncoding $transcriptFilePath | Should -BeExactly 'utf-16'
```

Reply:
Done